hamiltonia/Notepads
# update to latest sdk - code cleanup

- update to the latest sdk
- cleanup activation path to match existing patterns
- enable settings widget, no longer host in mainframe
- disable menu entries that don't make sense (full screen, etc.)
